Gemfile

```diff
@@ -1,7 +1,5 @@
-raise "Ruby versions < 3.0.1 are unsupported!"  if RUBY_VERSION < "3.0.1"
-warn "Ruby versions >= 3.2.0 are untested!" if RUBY_VERSION >= "3.2.0"
-raise "Ruby versions >= 3.4.0 are unsupported!" if RUBY_VERSION >= "3.4.0"
-
+ruby ">= 3.1.1", "< 3.5.0"
+warn "Ruby versions >= 3.4.0 are untested!" if RUBY_VERSION >= "3.4.0"
```

Comment:
Does this resolve the issue around rubocop and warning for testing of untested rubies?

Reply:
Yes. adding the padding on the top ( `< 3.5` instead of `< 3.4`) with the warning in there will help us in the future too.